sparse/mlir_backend/tests/test_simple.py

```diff
@@ -323,13 +300,9 @@ def test_coo_3d_format(dtype):
 
 @parametrize_dtypes
 def test_sparse_vector_format(dtype):
-    format = sparse.levels.get_storage_format(
-        levels=(sparse.levels.Level(sparse.levels.LevelFormat.Compressed),),
-        order="C",
-        pos_width=64,
-        crd_width=64,
-        dtype=sparse.asdtype(dtype),
-    )
+    if sparse.asdtype(dtype) in {sparse.complex64, sparse.complex128}:
+        pytest.xfail("Heisenbug")
+    format = sparse.formats.Coo().with_ndim(1).with_dtype(dtype).build()
```

Comment:
It's confusing for me that SparseVector format is being constructed from `Coo` to be honest. As an end-user I would go for a `Csf().with_ndim(1)` but this one is dense, right?

How about disallowing `Csf` and `Coo` with 1-dimension? And have `Dense` (arbitrary dimensionality) and separate `SparseVector` (only 1-dimension)?

Reply:
Well, `Coo` is actually 1 compressed + n singleton and `Csf` is 1 dense + n compressed.

So by that logic -- `Csf().with_ndim(1)` would be dense, while `Coo().with_ndim(1)` is actually what we want. So this statement is actually quite consistent. Would you be happy with documenting this behaviour?